.circleci/config.yml

```diff
@@ -533,6 +546,9 @@ workflows:
       - build-nightly-buster-securedrop-workstation-svs-disp:
           requires:
             - build-nightly-buster-securedrop-log
+      - build-nightly-buster-securedrop-workstation-config:
+          requires:
+            - build-nightly-buster-securedrop-workstation-svs-disp
```

Comment:
Filed https://github.com/freedomofpress/securedrop-debian-packaging/issues/236 to follow-up on that svs-disp package